### PR TITLE
Add onnxruntime 1.1.0 to CI

### PR DESCRIPTION
**Pull request by xadupre**

### CODE REVIEW

File: tests_onnxmltools/test_lightgbm.py [hunk 1]
```diff
@@ -16,6 +16,11 @@
 from skl2onnx.common.data_types import (
     SequenceType, DictionaryType, Int64TensorType, StringTensorType
 )
+import onnxmltools  # noqa
```
Comment (prabhat00155) [inline tests_onnxmltools/test_lightgbm.py]: Why these imports?

Comment (xadupre) [reply tests_onnxmltools/test_lightgbm.py]: To investigate which import of onnxmltools is failing.

Comment (prabhat00155) [reply tests_onnxmltools/test_lightgbm.py]: I see, you are not looking to check this in?

Comment (xadupre) [reply tests_onnxmltools/test_lightgbm.py]: I'd like to understand why onnxmltools is failing.

**Pull request closed without merging**